calcom/cal.com
# fix: Request permissions for shared Office365/Outlook calendars

## What does this PR do?

This change request the Calendar.ReadWrite.Shared scope for the Outlook API so events van be added to shared calendars. This used to work without this scope, but somehow it stopped working.

- Fixes #17298
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)

## Mandatory Tasks (DO NOT REMOVE)

- [ X ] I have self-reviewed the code (A decent size PR without self-review might be rejected).

## How should this be tested?

- When using Office365 within an organization create a shared calendar and share it with the office user that is authorizing the oAuth request from cal.com.
- Create an event-type and under 'Advanced' select the shared calendar to create the events on
- Schedule an event

Before this fix:
- The event won't appear on the shared calendar in Outlook

After this fix:
- The event is created and visible in Outlook

This update requires re-authorization so existing users of the Office365Calendar app might need to remove the app and re-install it get into effect.
